File: 1. Two Sum/main.go
package main

import "fmt"

func twoSum(nums []int, target int) []int {
	for i, v1 := range nums {
		for j := i + 1; j < len(nums); j++ {
			if v1+nums[j] == target {
				return []int{i, j}
			}
		}
	}
	return nil
}

func main() {
	fmt.Println(twoSum([]int{2, 7, 11, 15}, 9)) // [0, 1]
}

File: 1. Two Sum/main.ts
function twoSum(nums: number[], target: number): number[] {
    for(let i = 0; i < nums.length; i++) {
        for(let j = i; j < nums.length; j++) {
            if(i!= j && nums[i] + nums[j] == target) {
                return [i,j]
            }
        }
    }
    return [];
};


function main() {
    console.log(twoSum([2,7,11,15], 9)); // [0,1]
}

